docs/api.md

```diff
@@ -53,13 +53,94 @@ In our experience, any system that is successful needs to grow and change as new
 
 What constitutes a compatible change and how to change the API are detailed by the [API change document](devel/api_changes.md).
 
+
 ## API versioning
 
-To make it easier to eliminate fields or restructure resource representations, Kubernetes supports multiple API versions, each at a different API path prefix, such as `/api/v1beta3`. These are simply different interfaces to read and/or modify the same underlying resources. In general, all API resources are accessible via all API versions, though there may be some cases in the future where that is not true.
+To make it easier to eliminate fields or restructure resource representations, Kubernetes supports multiple API versions, each at a different API path prefix, such as `/api/v1beta3`.
 
 We chose to version at the API level rather than at the resource or field level to ensure that the API presents a clear, consistent view of system resources and behavior, and to enable controlling access to end-of-lifed and/or experimental APIs.
 
-The [API and release versioning proposal](design/versioning.md) describes the current thinking on the API version evolution process.
+Note that API versioning and Software versioning are only indirectly related.  The [API and release
+versioning proposal](design/versioning.md) describes the relationship between API versioning and
+software versioning.
+
+**Please do try our beta features.  During the beta stage, we are able to make significant changes
+in interfaces based on your feedback. In later stages, changes may not be possible.** Since alpha
+features are under active development, you may wish to try alpha features in a test cluster, rather
+than in a production cluster.
+
+Different API versions imply different levels of stability and support:
+ - Alpha level:
+   - The version names contain `alpha` (e.g. `v1alpha1`).
+   - *Purpose*: sharing new features with developers and eager expert users, prior to:
+     - *complete implementation*: for instance, some API operations, CLI
+     commands, or UI support may not be implemented. 
+     - *thorough API review*: the object schema may change in a later software release, and it may be necessary
```

Comment:
schema and/or semantics

While APIs are being developed, we break them a number of times, across all API versions. Existing objects may no longer function the same / correctly.